data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -33,13 +60,6 @@
       ]
     }
   ],
-  "location": [
```

Comment:
As noted in the pr comment, I don't think it's worth including GPS coordinate locations. We could maybe have a mechanism to ingest this data from external authority files. But having a field for users to enter it distracts from the role of these place records as name authorities for SDP; we won't have geolocation data to add from the Sinai collection